.eslintrc.js

```diff
@@ -24,6 +24,15 @@ module.exports = {
         "jest/no-focused-tests": "error",
         "jest/no-identical-title": "error",
         "jest/prefer-to-have-length": "warn",
-        "jest/valid-expect": "error"
+        "jest/valid-expect": "error",
+        "react/jsx-filename-extension": [1, { "extensions": [".js", ".jsx"] }]
+    },
+    "settings": {
+        "import/resolver": {
+        "node": {
+            "paths": ["src"]
+        }
```

Comment:
This file isn't formatted.

Reply:
This is the only thing delaying this PR. Can you fix it now so it can be merged?

Reply:
let me work on it